Sources/TuistGenerator/Generator/BuildPhaseGenerator.swift

```diff
@@ -357,4 +364,32 @@ final class BuildPhaseGenerator: BuildPhaseGenerating {
             embedWatchAppBuildPhase.files?.append(pbxBuildFile)
         }
     }
+
+    func generateEmbedAppClipsBuildPhase(path: AbsolutePath,
```

Comment:
I'd add a unit test that ensrues that we are adding the build phase to the `PBXTarget` with the right attributues when the target has associated app clips.